### PR TITLE
documentation for cfg_panic

### PR DESCRIPTION
**Pull request by cchiw**
I'm working on stabilizing the cfg_panic feature [issue](https://github.com/rust-lang/rust/issues/77443).

### CODE REVIEW

**Comment by marioidival** [comment]
@cchiw thank you

File: src/error/abort_unwind.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# `abort` and `unwind`
+
+The previous section illustrates the error handling mechanism `panic`.  The `cfg_panic` feature makes it possible to execute different code depending on the panic strategy. The current values available are `unwind` and `abort`.
```
Comment (ehuss) [inline src/error/abort_unwind.md:3]: I would not use internal code-names for communicating with rust users.  Nothing in the language or other documentation should reference `cfg_panic`.  Instead I would word it something like "Different code paths can be conditionally compiled based on the `panic` setting." or something like that.

File: src/error/abort_unwind.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# `abort` and `unwind`
+
+The previous section illustrates the error handling mechanism `panic`.  The `cfg_panic` feature makes it possible to execute different code depending on the panic strategy. The current values available are `unwind` and `abort`. 
+
+
+Building on the prior lemonade example, we explicitly use the panic strategy to execise different lines of code.  
+
+```rust,editable,ignore,mdbook-runnable
```
Comment (ehuss) [inline src/error/abort_unwind.md:8]: I'm not sure why this is `ignore`, but it means that the code won't be validated.

File: src/error/abort_unwind.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+fn main() {
+    drink("water");
+    drink("lemonade");
+}
+```
+
+Here is another example focusing on rewriting `drink()` and explicitly use the `unwind` keyword.
+
+```rust,editable,ignore
```
Comment (ehuss) [inline src/error/abort_unwind.md:27]: Same ignore here.

File: src/error/abort_unwind.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+fn main() {
+    drink("water");
+    drink("lemonade");
+}
+```
+
+The panic strategy can be set from the command line by using `abort` or `unwind`.
+
+```rust,editable,ignore
```
Comment (ehuss) [inline src/error/abort_unwind.md:48]: This isn't `rust` syntax.  It should be something like `shell` or `console`.  Also, it should not be `editable` or `ignore`.

File: src/error/abort_unwind.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+rustc  lemonade.rc -C panic=abort
```
Comment (ehuss) [inline src/error/abort_unwind.md:49]: I'm not sure what the `.rc` extension is. Perhaps this is meant to be `.rs`?

**Review by ehuss — COMMENTED** [review]
This should not have been merged before the feature was accepted and stabilized on master. This means the documentation will get out-of-sync with what is actually implemented.  I may need to skip updating rustc-by-example to avoid pulling this in.

File: src/error/abort_unwind.md (hunk 1 above)
Comment (cchiw) [reply src/error/abort_unwind.md:3]: Thanks for this feedback

File: src/error/abort_unwind.md (hunk 2 above)
Comment (cchiw) [reply src/error/abort_unwind.md:8]: Fixed in PR #1505

File: src/error/abort_unwind.md (hunk 4 above)
Comment (cchiw) [reply src/error/abort_unwind.md:48]: Fixed in PR #1505

**Pull request merged**